WORKSPACE

```diff
@@ -0,0 +1,69 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022 by Rivos Inc.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# Rivos salus (hypervisor) workspace
+#
+
+workspace(name = "salus")
+
+#
+# Rivos rules, repositories, and toolchains
+#
+
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+git_repository(
+    name = "rules_rivos",
+    remote = "git@gitlab.ba.rivosinc.com:rv/sw/int/bazel_rules_rivos.git",
```

Comment:
This dep means that RoW can't actually build with this today. Are there specific items we can bring into this project to avoid that?

Reply:
Waiting for a version of this to move out to github

Reply:
fixed